src/wp-includes/theme.php

```diff
@@ -866,6 +866,13 @@ function switch_theme( $stylesheet ) {
 
 	update_option( 'theme_switched', $old_theme->get_stylesheet() );
 
+	// Set autoload=no for the previous theme
```

Comment:
Missing period after inline comment.
```suggestion
	// Set autoload=no for the previous theme.
```